.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Hey Andrew, 

This file needs to be removed. It tells VSCode where python is and is specific to you and your machine. 
If you don't remember the command it's 
`git rm -r --cached .vscode`

Reply:
The issue has been corrected. Any other issues that you notice?